x-pack/plugins/case/common/constants.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const APP_ID = 'case';
+
+/**
+ * Case routes
+ */
+
+export const CASE_URL = '/api/cases';
```

Comment:
I think this should be `CASES_URL` since the endpoints that call it return the details for all cases. Then maybe the one below should be `CASE_URL` that would be consistant with the calls `getCases`, `getCase`